src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -54,6 +54,11 @@ public static boolean isCloudStorageUrl(final String path) {
         return path.startsWith(GCS_PREFIX);
     }
 
+    public static boolean isCloudStorageUrl(final java.nio.file.Path path) {
+        // the initial "" protects us against a null scheme
+        return ("" + path.toUri().getScheme() + "://").equals(GCS_PREFIX);
```

Comment:
I think this will still explode when `getScheme()` returns null, since the concatenation of a String with null causes a `NullPointerException` to be thrown. Need something like:

```
final String scheme = path.toUri().getScheme();
return scheme == null ? false : scheme + "://".equals(GCS_PREFIX);
```